deployment/ccip/changeset/v1_5_1/cs_add_token_e2e.go

```diff
@@ -572,6 +589,49 @@ func deployTokens(e cldf.Environment, tokenDeployCfg map[uint64]DeployTokenConfi
 					}
 				}
 			}
+			tokenAddresses[selector] = token.Address
+		case shared.BurnMintERC20Token:
+			token, err := cldf.DeployContract(e.Logger, e.BlockChains.EVMChains()[selector], ab,
+				func(chain cldf_evm.Chain) cldf.ContractDeploy[*burn_mint_erc20.BurnMintERC20] {
+					if cfg.MaxSupply == nil {
+						cfg.MaxSupply = big.NewInt(0)
```

Comment:
When would we want max supply to be zero? I could see this being confusing, someone using this & accidentally deploying a token with no possible supply.